apps/meteor/app/apps/server/bridges/persistence.ts

```diff
@@ -21,7 +20,8 @@ export class AppPersistenceBridge extends PersistenceBridge {
 			throw new Error('Attempted to store an invalid data type, it must be an object.');
 		}
 
-		return this.orch.getPersistenceModel().insertOne({ appId, data });
+		// #TODO: #AppsEngineTypes - Remove explicit types and typecasts once the apps-engine definition/implementation mismatch is fixed.
+		return this.orch.getPersistenceModel().insertOne({ appId, data }) as Promise<unknown> as Promise<string>;
```

Comment:
cast from InsertOneResult to string.

Reply:
Definitely a problem. I'm guessing at some point `insertOne` returned just the inserted id? Either that or another function was being used here and changed.